ui/components/multichain/account-list-menu/account-list-menu.tsx

```diff
@@ -410,11 +456,34 @@ export const AccountListMenu = ({
               }}
               ///: BEGIN:ONLY_INCLUDE_IF(multi-srp)
               selectedKeyringId={selectedKeyringId}
-              onSelectSRP={() => setActionMode(ACTION_MODES.SELECT_SRP)}
+              onSelectSRP={() => {
+                setPreviousActionMode(ACTION_MODES.ADD);
+                setActionMode(ACTION_MODES.SELECT_SRP);
+              }}
               ///: END:ONLY_INCLUDE_IF(multi-srp)
             />
           </Box>
         ) : null}
+        {clientType && chainId ? (
+          <Box paddingLeft={4} paddingRight={4} paddingBottom={4}>
+            <CreateSnapAccount
+              onActionComplete={async (confirmed: boolean) => {
```

Comment:
Should we use `useCallback` here?

Reply:
Yep, good idea, we can also reuse this function in few places